core/src/saros/session/internal/SarosSession.java

```diff
@@ -164,6 +167,19 @@ public void send(List<User> recipients, IActivity activity) {
 
         @Override
         public void execute(IActivity activity) {
+          /*
+           * Filters out resource activities for disposed projects
```

Comment:
What are disposed projects when translating this statement to Core context ?

Reply:
My bad. Missed that when removing the "disposed" references from the core stuff.